subiquity/common/types/storage.py

```diff
@@ -406,5 +406,6 @@ class ReformatDisk:
 
 @attr.s(auto_attribs=True)
 class EntropyResponse:
-    entropy: float
-    minimum_required: float
+    entropy_bits: int
+    min_entropy_bits: int
+    optimal_entropy_bits: int
```

Comment:
@d-loose I figured we haven't discussed in details what the API should return. 

Would you like to see:
* a boolean that tells if the check is a success or not?
* the list of "reasons" for a check to be a failure? (currently, this is only "low-entropy" I think)
* the message from snapd (e.g., "passphrase did not pass quality checks")

Anything else?